charts/flink-job/ci/full-values.yaml

```diff
@@ -75,26 +75,6 @@ job:
   initialSavepointPath: ""
   allowNonRestoredState: false
 
-podTemplate: |-
```

Comment:
Are these removals because of leftovers from something else? While I'm new to the space, it seems to be a lot of deletions considering the relatively small changes to the chart.

Reply:
Yes, they weren't meant to be used. Anything in `charts/*/ci` is used for testing, and these `podTemplate` entries broke the tests